src/main/resources/termsOfService.md

```diff
@@ -1,3 +1,5 @@
+# Terra Terms of Service
```

Comment:
The UI currently adds this header to the text that Sam returns, if Sam also returns this, then it'll have the header twice. Do we need Sam to return this for other portals? Otherwise, I'd say we should remove this here and leave it to the UI to display the header

Reply:
Yeah that may be the correct thing to do. I may have taken "verbatim" a bit too literally when copying over the entire doc. As long as the title is displayed when we present it to users, then we're good. Most portals will redirect to Terra UI, and those that won't (AoU) will be providing context for what it is

Reply:
Also looking at the revision history for the source document, it doesn't look like this title was explicitly added in this version of the text either